change/@fluentui-web-components-98f977e5-14f0-4053-83aa-f055df131551.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
@Hotell technically this is a dependency which impacts the distributed package - should this actually be _patch_?

Reply:
there is no patch in this branch only-prerelease. but you're right. this was merged as part of wc-3 branch update from master